pyproject.toml

```
@@ -11,7 +11,7 @@ keywords = ["eeg", "datasets", "reproducibility", "bci", "benchmark"]
 license = "BSD-3-Clause"
 
 [tool.poetry.dependencies]
-python = ">=3.9"
+python = ">=3.10,<4.0"
```

```suggestion
python = ">=3.9"
```
It seems that the `pyproject.toml` file was modified when you were trying to setup your environment using poetry. The `poetry.lock` file was also commited, while it should not. 
Can you remove `poetry.lock` and restore the previous state of `pyproject.toml` ?